tools/optimize.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Utility for optimizing ONNX models.
+
+Usage:
+    python optimize.py model.onnx optimized_model.onnx
+"""
+
+import argparse
+import os
+
+import onnx
+import onnx.inliner
+
+import onnxscript
+
+
+def main(args) -> None:
+    path = args.path
+    output_path = args.output_path
+
+    model = onnx.load(path, load_external_data=False)
+    # Hack: Change the working directory to the model directory so the optimizer
+    # can load external data files with relative paths.
+    # TODO: Remove this hack by fixing the optimizer to handle external data files properly.
+    pwd = os.getcwd()
+    model_dir = os.path.dirname(path)
+    os.chdir(model_dir)
+    model = onnxscript.optimizer.optimize(model)
+    model = onnx.inliner.inline_local_functions(model)
```

Comment:
Is this necessary?

Reply:
Since the current optimizer will specialize functions and retain them I wanted to flatten everything with this